pkg/console/subresource/configmap/configmap.go

```diff
@@ -53,6 +53,7 @@ func DefaultConfigMap(
 		OAuthServingCert(useDefaultCAFile).
 		APIServerURL(getApiUrl(infrastructureConfig)).
 		InactivityTimeout(inactivityTimeoutSeconds).
+		ManagedClusterConfigFile(fmt.Sprintf("%v/%v", api.ManagedClusterConfigMountDir, api.ManagedClusterConfigKey)).
```

Comment:
```suggestion
		ManagedClusterConfigFile(fmt.Sprintf("%s/%s", api.ManagedClusterConfigMountDir, api.ManagedClusterConfigKey)).
```

Reply:
Is this file always present on the cluster ? What does it contain if the cluster is not part of the ACM ?

Reply:
Wondring if we should not pass `canMountManagedClusterConfig` to the `DefaultConfigMap()` method and based on that bool either pass the `fmt.Sprintf("%s/%s", api.ManagedClusterConfigMountDir, api.ManagedClusterConfigKey)` or not?